src/bolt/utils.ts

```diff
@@ -0,0 +1,93 @@
+import {
+    ReactionAddedEvent,
+    ReactionRemovedEvent
+} from '@slack/bolt';
+import {
+    ReactionData,
+    UserInfo
+} from './types';
+import {
+    ConversationsRepliesResponse
+} from '@slack/web-api/dist/response/ConversationsRepliesResponse';
+import {
+    GenericMessageEvent
+} from '@slack/bolt/dist/types/events/message-events';
+import app from './app';
+
+let botUserId : string | unknown = null;
```

Comment:
is unknown the right type here? is this more appropriate?

```suggestion
let botUserId : string | null;
```